src/DocConfiguration.ts

```diff
@@ -0,0 +1,164 @@
+import express from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+const swaggerDefinition = {
+    openapi: '3.0.0',
+    info: {
+        title: 'RIF Relay Server API',
+        version: '1.0.0',
+        description: 'This is a API application provided to support RIF Relay.',
+        license: {
+            name: 'Licensed Under MIT',
+            url: 'https://spdx.org/licenses/MIT.html'
+        },
+        contact: {
+            name: 'RSK Smart',
+            url: 'https://developers.rsk.co/'
```

Comment:
I preferred to point to the developer portal since that I assume the users will be mostly developers.